go/websocketbridge/features/serverToClient.feature

```diff
@@ -0,0 +1,37 @@
+Feature: 
+
+  As an Exosphere developer
+  I want to be able to send messages from my Exosphere application to a client
+  So that I can create a useful application
+
+  Rules
+  -
```

Comment:
```
- messages received from exorelay are forwarded to the client if and only if the session id matches
```
Couple tests to add here would be: 
* a message is received with no session id, ensure it isn't sent to any clients
* a message is received with a session id, check it isn't sent to other clients